app/javascript/dashboard/components/layout/sidebarComponents/AccountSelector.vue

```diff
@@ -69,10 +69,7 @@ export default {
       </div>
     </div>
 
-    <div
-      v-if="globalConfig.createNewAccountFromDashboard"
-      class="flex items-center justify-end gap-2 px-8 pt-4 pb-8"
-    >
+    <div class="flex items-center justify-end gap-2 px-8 pt-4 pb-8">
```

Comment:
Please bring back the config. The button is for internal testing only, we have not given this option to anyone.

Reply:
This got slipped in accidentally, will add it back